.github/workflows/pr.yml

```
@@ -16,7 +16,7 @@ jobs:
         os: [windows-2019, ubuntu-latest, macos-latest]
         # list only the earliest and latest node versions supported
         # this makes PR builds more efficient
-        node-version: [14, 16]
+        node-version: [14, 16, 18]
```

If I read the comment correctly, this should be:

```suggestion
        node-version: [14, 18]
```

---

Updated, thanks. Yea, I clearly missed that.